cfssl.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: cfssl
+  version: 1.6.4
+  epoch: 0
+  description: Cloudflare's PKI and TLS toolkit
+  copyright:
+    - license: BSD-2-Clause
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cloudflare/cfssl
+      tag: v${{package.version}}
+      expected-commit: b4d0d877cac528f63db39dfb62d5c96cd3a32a0b
+
+  - runs: make all
```

Comment:
This uses `-s`: https://github.com/cloudflare/cfssl/blob/master/Makefile#L2

Reply:
I know, I haven't brought myself to start patching it out of things